testing.md

```diff
@@ -0,0 +1,91 @@
+# How to test changes
+This document assumes you will be using personal quay.io registry to push bundle and operator images to avoid accidentally changing production registries
+```
+export QUAY_REGISTRY=quay.io/<your_namespace>
+```
+
+Also create a CatalogSource
+```
+cat <<EOF | tee >(oc apply -f -) | cat
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: test-cs-operator
+  namespace: openshift-marketplace
+spec:
+  displayName: test-cs-operator
+  publisher: IBM
+  sourceType: grpc
+  image: quay.io/opencloudio/ibm-common-service-catalog:3.7.0-beta
+  updateStrategy:
+    registryPoll:
+      interval: 45m
+EOF
+```
+
+If pushing an image to your quay.io registry with its first tag, the image repository may be set to private by default, so you will need to login to quay.io and change the repository settings to public in order for your cluster to pull the image.
+
+The actual testing consist of:
+1. verify common-service operator pod is Running
+2. verify namespace scope operator and ODLM operator are installed
+3. verify secretshare and common-service-webhook pods are running
+4. verify functionality of common-service operator such as:
+    - size profile
+
+## Test fresh installation with bundle
+1. add features/fixes to repo
+2. change `image` value in config/manager/manager.yaml to `quay.io/<your_namespace>/common-service-operator:dev`
+   - and any of the operand image values if necessary
+3. build operator with changes
+```
+make build-dev
+```
+4. build bundle containing changes and bundle image
+```
+make bundle-manifests VERSION=99.99.99
```

Comment:
please avoid version `99.99.99` as we'd like to use this version as a magic version for feature branches

Reply:
the 99.99.99 is just a way to signify that this bundle is newer than what was just deployed. The `VERSION` could be anything as long as it follows semantic versioning and is higher than the previous version. During development, it is also expected that developer will use personal namespace for testing, so nothing should be pushed to `opencloudio`